lib/fresh_service/resources/asset.ex

```diff
@@ -0,0 +1,40 @@
+defmodule FreshService.Asset do
+  @moduledoc """
+  Represents an Asset resource in the FreshService API.
+  - [FreshService docs](http://api.freshservice.com/#asset)
+  """
+
+  import FreshService.Api, only: [get: 2]
+
+  defstruct agent_id: nil,
+            agent_name: nil,
+            asset_tag: nil,
+            assigned_on: nil,
+            business_impact: nil,
+            ci_type_id: nil,
+            ci_type_name: nil,
+            created_at: nil,
+            department_id: nil,
+            department_name: nil,
+            depreciation_id: nil,
+            description: nil,
+            display_id: nil,
+            id: nil,
+            impact: nil,
+            levelfield_values: nil,
+            location_id: nil,
+            location_name: nil,
+            name: nil,
+            product_name: nil,
+            salvage: nil,
+            state_name: nil,
+            updated_at: nil,
+            usage_type: nil,
+            used_by: nil,
+            user_id: nil,
+            vendor_name: nil
+
+  def all(page \\ 1) do
+    get("cmdb/items.json?page=#{page}", __MODULE__)
```

Comment:
I guess in future we would like to support more params. For now it's fine I think.

Reply:
What params do you have in mind? I don't see any more available in the FreshService API documentation for this endpoint

Reply:
Okk, what I want to say is that freshservce endpoints might support multiple params (let's say, `per_page` and `filter`) and we can expose a more explicit api client.
For example
```elixir
FreshService.Resources.Asset.all(page: 4, per_page: 50, filter: "android")
```
imho looks better than
```elixir
FreshService.Resources.Asset.all(4, 50, "android")
```

Reply:
Good point 👍